trin-validation/src/header_validator.rs

```diff
@@ -226,82 +226,21 @@ mod test {
     use alloy_primitives::{Address, Bloom, B256, U256};
     use alloy_rlp::Decodable;
     use rstest::*;
-    use serde_json::{json, Value};
+    use serde_json::Value;
     use ssz::{Decode, Encode};
-    use tokio::sync::mpsc;
     use tree_hash::TreeHash;
 
     use crate::constants::DEFAULT_PRE_MERGE_ACC_HASH;
     use ethportal_api::{
-        types::{
-            execution::{
-                accumulator::EpochAccumulator,
-                header_with_proof::{
-                    BlockHeaderProof, HeaderWithProof, PreMergeAccumulatorProof, SszNone,
-                },
+        types::execution::{
+            accumulator::EpochAccumulator,
+            header_with_proof::{
+                BlockHeaderProof, HeaderWithProof, PreMergeAccumulatorProof, SszNone,
             },
-            jsonrpc::{endpoints::HistoryEndpoint, request::HistoryJsonRpcRequest},
         },
         utils::bytes::{hex_decode, hex_encode},
-        HistoryContentKey, RawContentKey,
     };
 
-    #[rstest]
-    #[case(1_000_001)]
-    #[case(1_000_002)]
-    #[case(1_000_003)]
-    #[case(1_000_004)]
-    #[case(1_000_005)]
-    #[case(1_000_006)]
-    #[case(1_000_007)]
-    #[case(1_000_008)]
-    #[case(1_000_009)]
-    #[case(1_000_010)]
-    #[tokio::test]
-    async fn generate_and_verify_header_with_proofs(#[case] block_number: u64) {
```

Comment:
This test still seems somewhat useful to leave in. We still want to test that trin constructs & validates (and invalidates) these header with proofs...
I would just update this test (& `invalidate_invalid_proofs`) to lookup the epoch accs directly from disk and use `PreMergeAccumulator::construct_proof()` instead of the removed `PreMergeAccumulator::generate_proof()`